app-config.yaml

```diff
@@ -216,6 +216,15 @@ catalog:
         - Domain
         - Location
 
+  providers:
```

Comment:
Should probably remove this from here - especially since most people out there probably won't have the AZURE_TENANT_ID env var set.

As a hint if this is a development remnant, you can always stash stuff like that in app-config.local.yaml which isn't source controlled